src/test/Testcafe/runtime_tests/runtime-tests.js

```diff
@@ -33,6 +33,7 @@ test('Instantiate an app in runtime', async () => {
     .expect(runtime.startNewButton.exists).ok()
     .click(runtime.startNewButton)
     .switchToMainWindow()
+    .wait(5000)
```

Comment:
it is better to use 'timeout' in an expect statement rather than an explicit 'wait'.

Reply:
Left over from testing. Wait isn't needed in this test. Removed.